misc/python/materialize/cli/mzbench.py

```diff
@@ -79,11 +81,12 @@ def main(args: argparse.Namespace) -> None:
     else:
         git_references = [None, *args.git_references]
 
-    if args.verbose:
-        build_tags = [None, *[mzbuild_tag(ref) for ref in args.git_references]]
-        print(f"DEBUG: num_iterations={args.num_measurements}")
-        print(f"DEBUG: worker_counts={worker_counts}")
-        print(f"DEBUG: mzbuild_tags={build_tags}")
+    ui.Verbosity.init_from_env(explicit=args.quiet)
+    dbg = ui.speaker("DEBUG: ")
```

Comment:
tiny nit: I believe style is to make this a module var so that it's easy to reuse elsewhere if desired.